addons/web/static/src/views/form/form_controller.scss

```diff
@@ -240,9 +240,13 @@
     // No sheet
     &.o_form_nosheet, .o_form_nosheet {
         @include o-webclient-padding($top: $o-sheet-vpadding, $bottom: $o-sheet-vpadding);
+        background-color: var(--formView-sheet-background);
 
         .o_form_statusbar {
             margin: (-$o-sheet-vpadding) (-$o-horizontal-padding) $o-sheet-vpadding (-$o-horizontal-padding);
+            background-color: map-get($grays, '100');
```

Comment:
I`d use the `$o-webclient-background-color` variable instead ... it's same color, but your code would be easier to understand (and if one day we change the webclient's bg, then the new value will be correctly applied here too ;) )